From 2a769d0843947b44b107807af75e682404efa4dd Mon Sep 17 00:00:00 2001
From: Hu hui <b29976@freescale.com>
Date: Tue, 12 Oct 2010 14:05:13 +0800
Subject: [PATCH] ENGR00132477-2 IMX USB:fix some wakeup releated issues

Common Part
move the is_b_host detecting from common code to the driver,so keep
common code be same as the community

Signed-off-by: Hu Hui <b29976@freescale.com>
---
 drivers/usb/core/driver.c |    7 -------
 1 files changed, 0 insertions(+), 7 deletions(-)

diff --git a/drivers/usb/core/driver.c b/drivers/usb/core/driver.c
index 52aad63..babc9a2 100644
--- a/drivers/usb/core/driver.c
+++ b/drivers/usb/core/driver.c
@@ -1313,13 +1313,6 @@ int usb_resume(struct device *dev, pm_message_t msg)
 	struct usb_device	*udev = to_usb_device(dev);
 	int			status;
 
-	/*
-	 * At otg mode, it should not call host resume for usb gadget device
-	 * Otherwise, this usb device can't be recognized as a gadget
-	 */
-	if (udev->bus->is_b_host)
-		return 0;
-
 	/* For PM complete calls, all we do is rebind interfaces */
 	if (msg.event == PM_EVENT_ON) {
 		if (udev->state != USB_STATE_NOTATTACHED)
-- 
1.5.4.4

